lib/http.ex

```diff
@@ -38,6 +38,10 @@ defmodule Braintree.HTTP do
     {"Content-Type", "application/xml"}
   ]
 
+  @timeout Braintree.get_env(:timeout, 8000) # HTTPoison default
```

Comment:
Module attributes are set during compile time, I think this will always be 8000 regardless of what the environment is (because the application hasn't booted). What I meant was to put `@timeout 8000` and then reference it below.